subsys/ipc/rpmsg_multi_instance/Kconfig

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2021 Nordic Semiconductor (ASA)
+# SPDX-License-Identifier: Apache-2.0
+
+# Workaround for not being able to have commas in macro arguments
+DT_CHOSEN_Z_IPC_SHM := zephyr,ipc_shm
+
+menuconfig RPMSG_MULTI_INSTANCE
+	bool "RPMsg multiple instance"
+	select IPM
+	select OPENAMP
+	help
+	  Enables support for RPMsg multiple instance.
+
+if RPMSG_MULTI_INSTANCE
+
+choice RPMSG_ROLE
+	prompt "RPMSG device role"
+	default RPMSG_MULTI_INSTANCE_REMOTE
+
+config RPMSG_MULTI_INSTANCE_REMOTE
+	bool "Remote"
+
+config RPMSG_MULTI_INSTANCE_MASTER
+	bool "Master"
+endchoice
+
+config RPMSG_MULTI_INSTANCES_NO
+	int "Number of RPMSG instances."
```

Comment:
Should it be defined range now to align with the number of defined instances below?

Reply:
Right. Range has been added.